storage_server.repo.in

```diff
@@ -11,13 +11,13 @@ enabled_metadata=1
 
 [lustre]
 name=Lustre Server
-baseurl=https://downloads.whamcloud.com/public/lustre/lustre-2.10.4/el7/server/
+baseurl=https://downloads.whamcloud.com/public/lustre/lustre-2.10.5/el7/server/
```

Comment:
Would it make sense to make the lustre version an environment variable? The reason I ask is because this could be set as a CI environment variable and then we wouldn't need to update specific files in the future. We might even be able to add a hook such that when lustre is released it will automatically update the environment variable.

Reply:
> Would it make sense to make the lustre version an environment variable? The reason I ask is because this could be set as a CI environment variable and then we wouldn't need to update specific files in the future.

There is already an override for this file iirc, which is why it's a .in.

I suppose we could extract it to an env var, but it's not like it's in terribly many places right now anyway.

> We might even be able to add a hook such that when lustre is released it will automatically update the environment variable.

This could be done with probot + glitch, but given the frequency we need to do this, not sure the ROI is there at this point

Reply:
👍